test/integration/test_kraken.py

```diff
@@ -92,20 +128,21 @@ def setUpClass(cls):
         cls.bam = cls.input_bam('zaire_ebola')
 
     def test_kraken_tool(self):
-        output = os.path.join(tempfile.tempdir, 'zaire_ebola.kraken')
-        output_filtered = os.path.join(tempfile.tempdir, 'zaire_ebola.filtered-kraken')
-        output_report = os.path.join(tempfile.tempdir, 'zaire_ebola.kraken-report')
-        self.assertEqual(0, self.kraken.classify(self.db, self.fastqs, output).returncode)
+        out = join(tempfile.tempdir, 'zaire_ebola.kraken')
```

Comment:
I think you need to create a mkdtemp instead of just dropping things straight in /tmp